src/api/config/initializers/git.rb

```diff
@@ -1,6 +1,6 @@
 module Git
   if Rails.root.join('last_deploy').exist?
-    COMMIT = Rails.root.join('last_deploy').open('r') { |f| GIT_REVISION = f.gets.try(:chomp) }
+    COMMIT = Rails.root.join('last_deploy').open('r') { |f| const_set(:GIT_REVISION, f.gets.try(:chomp)) }
```

Comment:
@eduardoj I think there is no point of using a constant for `GIT_REVISION` at all. Not used anywhere.

Reply:
That's correct. I removed the assignment. Thanks!